README.md

```diff
@@ -1,9 +1,39 @@
+[comment]: # (NOTE: This file is generated and should not be modify directly. Update `templates/ROOT_README.hbs.md` instead)
 # Web foundation
 
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)
+[![Build Status](https://travis-ci.org/Shopify/web-foundation.svg?branch=master)](https://travis-ci.org/Shopify/web-foundation)
+[![codecov](https://codecov.io/gh/Shopify/web-foundation/branch/master/graph/badge.svg)](https://codecov.io/gh/Shopify/web-foundation)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
+
 This repo is the living guide to how we build web UI at Shopify. It captures the [principles](Principles) we use to make decisions, our [best practices](Best%20practices) and [styleguides](Styleguides) for writing UI code, and a [log of decisions](Decision%20records) that led us to this foundation. It also provides a list of [resources](Resources) for mastering the different parts of web UI development.
```

Comment:
The web-foundation core docs are a bit scattered across the root directory now. I think we should all put them under on directory so there easier to navigate. Perhaps a directory called `web-foundation`. We can also move the older readme there and leave a like in the root redme to the `web-foundation/README.md`

Reply:
What about all in the wiki?

Reply:
I'm not sure `web-foundation` is a good name for that folder. As a whole it sort of forms a bigger styleguide so I'd be tempted to call it that, but then it's weird to have a styleguides folder inside that.

Maybe "web handbook"? :P

Reply:
What about moving every documentation into the wiki :) ?

Reply:
I was gonna suggest `documentation` ot `docs`

Reply:
Ok let's move everything undo `handbook` since `docs` is already used? Everyone is good with this?